lib/device.js

```diff
@@ -177,8 +179,12 @@ class Device extends EventEmitter {
   getReceiverStatus (callback) {
     if (!callback) callback = noop
 
-    this._tryConnect(() => {
-      this.client.getStatus(callback)
+    this._tryConnect((err) => {
+      if (err !== null) {
+        this.client.getStatus(callback)
+      } else {
+        callback(err)
+      }
```

Comment:
Shouln't this be like this?

```suggestion
      if (err) return callback(err)
      this.client.getStatus(callback)
```